subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/internal/EclipseProjectMetadata.java

```diff
@@ -56,4 +56,8 @@ public File getFile() {
     public Set<? extends Task> getGeneratorTasks() {
         return Collections.singleton(generatorTask.get());
     }
+
+    public boolean hasJavaTestFixtures() {
+        return eclipseModel.getClasspath().getContainsTestFixtures().getOrElse(false);
```

Comment:
I think it would be nicer if we always set a convention and only do `get()` here. Which means adding `containsTestFixtures.convention(false)` in the `EclipseClasspath ` constructor.

You may change it if you agree (or leave it as it is).

Reply:
Yeah, that makes sense; I've pushed the implementation.